.github/workflows/sync-automation.yml

```diff
@@ -41,13 +38,46 @@ jobs:
           token: ${{ secrets.ORG_GH_TOKEN }}
 
       - name: sync ${{matrix.branch}} from master
+        id: sync-changes
         run: |
-          git fetch origin master
           git config --local user.email "wf-gen@tyk-ci"
           git config --local user.name "Bender"
-          for f in $FILES; do 
-            git checkout --theirs origin/master -- $f || true
-          done
-          git commit -a -m "[CI] Syncing release automation from master"
-          git push origin
-          echo "::debug::${FILES} syncd for ${{matrix.branch}}"
+          git fetch origin ${{ matrix.branch }}
+          prbranch=sync/auto/${{ matrix.branch }}/${{ github.run_id }}
+          git checkout -b $prbranch
```

Comment:
Better to parametrise this. Unless it is a template variable.

Reply:
Was thinking of making them template vars.  The same set of files are used in three different places - the trigger condition, deletion, and restore. These might also differ from repo to repo. We could rewrite the template accordingly, so it's handled there.